src/libmongoc/tests/test-mongoc-change-stream.c

```diff
@@ -2774,6 +2773,6 @@ test_change_stream_install (TestSuite *suite)
    TestSuite_AddMockServerTest (
       suite, "/change_streams/prose_test_18", prose_test_18);
 
-   test_framework_resolve_path (JSON_DIR "/change_streams/legacy", resolved);
-   install_json_test_suite (suite, resolved, &test_change_stream_spec_cb);
+   install_json_test_suite (
+      suite, JSON_DIR, "change_streams", &test_change_stream_spec_cb);
```

Comment:
I believe this is the cause of the test failures. This should include the `legacy` directory appended.

Reply:
Good find. I don't know how I missed that one.